dali/imgcodec/decoders/nvjpeg2k/nvjpeg2k_helper.h

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2020-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright (c) 2020-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
no changes after all, please revert the copyright header changes in this file

Reply:
It was changed:

`diff dali/imgcodec/decoders/nvjpeg/nvjpeg_helper.h dali/operators/decoder/nvjpeg/nvjpeg_helper.h`
```diff 
15,16c15,16
< #ifndef DALI_IMGCODEC_DECODERS_NVJPEG_NVJPEG_HELPER_H_
< #define DALI_IMGCODEC_DECODERS_NVJPEG_NVJPEG_HELPER_H_
---
> #ifndef DALI_OPERATORS_DECODER_NVJPEG_NVJPEG_HELPER_H_
> #define DALI_OPERATORS_DECODER_NVJPEG_NVJPEG_HELPER_H_
39d38
< namespace imgcodec {
100a100,119
> template <>
> inline void cudaResultCheck<nvjpegStatus_t>(nvjpegStatus_t status) {
>   switch (status) {
>   case NVJPEG_STATUS_SUCCESS:
>     return;
>   default:
>     throw dali::NvjpegError(status);
>   }
> }
> 
> template <>
> inline void cudaResultCheck<nvjpegStatus_t>(nvjpegStatus_t status, const string &extra) {
>   switch (status) {
>   case NVJPEG_STATUS_SUCCESS:
>     return;
>   default:
>     throw dali::NvjpegError(status, extra.c_str());
>   }
> }
> 
221,242d239
< }  // namespace imgcodec
< 
< template <>
< inline void cudaResultCheck<nvjpegStatus_t>(nvjpegStatus_t status) {
<   switch (status) {
<   case NVJPEG_STATUS_SUCCESS:
<     return;
<   default:
<     throw imgcodec::NvjpegError(status);
<   }
< }
< 
< template <>
< inline void cudaResultCheck<nvjpegStatus_t>(nvjpegStatus_t status, const string &extra) {
<   switch (status) {
<   case NVJPEG_STATUS_SUCCESS:
<     return;
<   default:
<     throw imgcodec::NvjpegError(status, extra.c_str());
<   }
< }
< 
245c242
< #endif  // DALI_IMGCODEC_DECODERS_NVJPEG_NVJPEG_HELPER_H_
---
> #endif  // DALI_OPERATORS_DECODER_NVJPEG_NVJPEG_HELPER_H_
```

Reply:
Oops, I didn't notice that it was the nvjpeg2k file, not the nvjpeg one (whose diff i replied with in the previous comment).
I'll remove the copyright notice update.